libraries/chain/include/eosio/chain/block_state.hpp

```diff
@@ -83,6 +83,8 @@ struct block_state : public block_header_state {     // block_header_state provi
                deque<transaction_receipt>&& trx_receipts, const std::optional<quorum_certificate>& qc,
                const signer_callback_type& signer, const block_signing_authority& valid_block_signing_authority);
 
+   block_state(snapshot_detail::snapshot_block_state_v7&& sbs);
```

Comment:
Since it is a move constructor not a huge deal, but probably better to make `explicit`.

Reply:
`explicit` in constructors is useful to avoid unexpected conversions, say from an `int` to a `time_stamp`. 

However, with an argument type of  `snapshot_detail::snapshot_block_state_v7&&`, I don't think we have to worry too much about that.

Reply:
Still I'll make it explicit, I guess it doesn't hurt in this case.